docs/src/utils/HomeCallToAction.tsx

```
@@ -0,0 +1,40 @@
+import React from "react";
+import  ActionButton  from "./ActionButton";
+
+interface HomeCallToActionProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)